aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js

```diff
@@ -311,48 +344,181 @@ const WizardHeader = ({ active, setActive, setOpen }) => {
 };
 
 WizardHeader.propTypes = {
-  active: PropTypes.number.isRequired,
-  setActive: PropTypes.func.isRequired,
   setOpen: PropTypes.func.isRequired,
 };
 
+const Guidance = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
+      alignSelf="start"
+      background="background-contrast"
+      gap="medium"
+      pad="medium"
+      round="small"
+      flex
+      width={size !== 'small' ? { min: 'small' } : '100%'}
+    >
+      <Text color="text-strong" size="large">
+        When guidance is required for the form or content of the wizard, you
+        might consider a two-column format.
+      </Text>
+      <Box direction="row" gap="small">
+        <Stakeholder color="text-strong" />
+        <Text color="text-strong">Instruction for the first field.</Text>
+      </Box>
+      <Box direction="row" gap="small">
+        <ContactInfo color="text-strong" />
+        <Text color="text-strong">Instruction for the next field.</Text>
+      </Box>
+      <Box direction="row" gap="small">
+        <UserAdd color="text-strong" />
+        <Text color="text-strong">
+          Some information that helps to complete the next field.
+        </Text>
+      </Box>
+    </Box>
+  );
+};
+
+const StepHeader = () => {
+  const { activeIndex, activeStep } = useContext(WizardContext);
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box id="sticky-header-two-column" gap="medium" flex={false}>
+      <Box>
+        <Text color="text-strong">
+          Step {activeStep} of {steps.length}
+        </Text>
+        <Heading color="text-strong" margin="none">
+          {steps[activeIndex].title || `Step ${activeStep} Title`}
+        </Heading>
+      </Box>
+      {size === 'small' && (
+        <Text color="text-strong" size="large">
+          {steps[activeIndex].description}
+        </Text>
+      )}
+    </Box>
+  );
+};
+
+const StepFooter = () => {
+  const size = useContext(ResponsiveContext);
+  const {
+    activeIndex,
+    setActiveIndex,
+    formValues,
+    setError,
+    setAttemptedAdvance,
+  } = useContext(WizardContext);
+
+  const buttonProps = {
+    fill: size === 'small' ? 'horizontal' : undefined,
+    icon: <FormNextLink />,
+    primary: true,
+    reverse: true,
+  };
+
+  return (
+    <Box
+      pad={{
+        horizontal: size !== 'small' ? 'large' : undefined,
+      }}
+      flex={false}
+    >
```

Comment:
I think we should get rid of this outer box since Footer is a box itself the outer is extraneous.

I commented it out locally and seems fine, but need to verify. I don't believe the single column wizard has this and it looks good.

Reply:
This Box is needed as the screen shrinks to ensure the edges of the footer stay in line with the form content until the screen breakpoint of small is hit.